db/migrate/20181022222945_create_post.rb

```diff
@@ -0,0 +1,16 @@
+class CreatePost < ActiveRecord::Migration[5.1]
+  def change
+    create_table :posts do |t|
+
+      t.integer :user_id, 		 null: false 
+
+      ## Trackable
```

Comment:
I think this comment is copied from the `User` migration file - might be confusing to future developers